Tests/Configuration/appConfig.json

```diff
@@ -1,3 +1,5 @@
 {
-    "url": ""
+    "url": "",
+    "rnwAdminUrl":"",
+    "adminurl":""
```

Comment:
Is this variable used or is it duplicate of rnwAdminUrl?
```suggestion
    "adminurl":""
```

Reply:
there is separate feature branch , later it will merge